src/reaper_osara.cpp

```diff
@@ -1718,6 +1718,37 @@ void postSelectMultipleItems(int command) {
 	SetCursorContext(1, nullptr);
 }
 
+void postMoveItemOrEnvelopePoint(int command) {
+	TrackEnvelope* envelope = GetSelectedEnvelope(nullptr);
+	if (!envelope) {
+		int count = CountSelectedMediaItems(0);
+		if(count==0) {
+			outputMessage(translate("no items selected"));
+			return;
+		}
+		// Translators: Reported when moving items{} will be replaced by the
+	// number of items; e.g. "2 items moved".
+		switch(command) {
+			case 40117: {
+				outputMessage(format(
+					translate_plural("{} item moved up", "{} items moved up", count),
+					count));
+				break;
+			}
+			case 40118: {
+				outputMessage(format(
+					translate_plural("{} item moved down", "{} items moved down", count),
+					count));
+				break;
+			}
+//			default: {
+//				outputMessage("Unknown case ");
+//			}
+		} 
+	}
+	else postMoveEnvelopePoint(command);
```

Comment:
nit: else on same line as closing brace, avoid unbraced else block.
```suggestion
	} else {
		postMoveEnvelopePoint(command);
	}
```